ggml/src/ggml-sycl/element_wise.cpp

```diff
@@ -1,6 +1,10 @@
 #include "common.hpp"
 #include "ggml.h"
 #include "element_wise.hpp"
+#include <sycl/detail/builtins/builtins.hpp>
+#include <sycl/nd_item.hpp>
+#include <sycl/nd_range.hpp>
+#include <sycl/range.hpp>
```

Comment:
Why is the first header used? We should avoid relying on `detail` includes if possible.
It is more common to just include `<sycl/sycl.hpp>`.  They could also all be removed if you're happy to rely on previous includes.

Reply:
Right. These includes gets auto imported by neovim editor. I need to disable autoimport in nvim-lsp.

Will remove it tomorrow.